src/app/features/home/components/home-button/home-button.component.scss

```diff
@@ -0,0 +1,11 @@
+@import 'node_modules/govuk-frontend/govuk/all';
+
+.home-button {
+  background: govuk-colour('light-grey');
+  margin: 0 0 10px 18px;
+  width: 364px !important;
```

Comment:
nit: not a huge fan of important but unsure how reusable home button is as a button class